core/src/test/java/androidx/documentfile/provider/AssetsDocumentFile.java

```diff
@@ -0,0 +1,142 @@
+package androidx.documentfile.provider;
```

Comment:
Could you please move this to our own namespace, eg. `de.danoeh.antennapod.core.storage`? The only reason why we have some classes in foreign packages is that we implemented workarounds that require access to package-private members.

Reply:
That's exactly the reason why I had to use that package: The constructor of the super class `DocumentFile` is package-accessible only. If you prefer, I can change back that part of the test code and use Mockito instead.

Reply:
👍 I did not notice that the constructor is protected when having a quick look at the code.